Resources/Prototypes/Entities/Markers/Spawners/Random/paintings.yml

```
@@ -27,6 +27,10 @@
       - PaintingRedBlueYellow
       - PaintingHelloWorld
       - PaintingSpoon # DeltaV Painting, see Resources/Prototypes/_DV/Entities/Structures/Wallmount/painting.yml
+      - PaintingUnePipe #den port
+      - PaintingTheWrathOfGods #den port
+      - PaintingHumanCondition #den port
+      - PaintingKnowledge #den port
```

```suggestion
      - PaintingUnePipe # _DEN edit
      - PaintingTheWrathOfGods # _DEN edit
      - PaintingHumanCondition # _DEN edit
      - PaintingKnowledge # _DEN edit
```

More clear on comment.